src/main/docker/app.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
```

Comment:
I think, we can delete this file, it is no need (see the folder https://github.com/jhipster/jhipster-registry/tree/master/src/main/docker)
Do you confirm @PierreBesson ?